### ISSUE TITLE
Error: amp-iFrame is not working inside amp-selector inside an accordion

### ISSUE THREAD

**Issue by nitindumbre** [ampproject/amphtml#23449]
## What's the issue?

Error: amp-iframe is not working inside amp-selector inside an accordion

## How do we reproduce the issue?

iframe contents are not getting loaded when the iframe is inside amp-selector inside an accordion

If we used iframe inside the selector or accordion then the iframe content is getting loaded..

Please find the sample code . Go to the 'AMP iframe Selector' tab and see the iframe content is not getting loaded until unless we clicked on the iframe area.

![amp-iframe](https://user-images.githubusercontent.com/53178651/61645096-db43b000-ac9d-11e9-8b37-233ac6661cdd.png)

Sample Code [amp-selector-iframe.zip](https://github.com/ampproject/amphtml/files/3418152/amp-selector-iframe.zip)

**Comment by nainar** [comment]
Hi @cathyxz could you explore this? It seems like there is some issue with the loading detection. 

The iframe loads when you open devtools after page loads.

**Comment by nitindumbre** [comment]
Hi Team,

Did you get any chance to look into the issue ?

**Comment by romeuflores** [comment]
It is also worth noting that the same happens for an amp-list inside an amp-selector inside an amp-accordion.

**Comment by cathyxz** [comment]
I think this is an issue with `<amp-selector>` not laying out its children correctly when `display` is toggled via the `selected` attribute. I tested a few things: 

`<amp-iframe>`: 
- correctly lays out when display is [toggled via the `hidden` attribute](https://codepen.io/cathyxz/pen/MNvJZx)
- correctly lays out when display is [toggled via `toggleClass` action](https://codepen.io/cathyxz/pen/BXJNNP)

Whereas the original bug repros without an `<amp-accordion>`: https://codepen.io/cathyxz/pen/XvVXVw?editors=1000.

**Comment by romeuflores** [comment]
Hi Cathy,

Our reproduction doesn't involve toggling/toggle actions.

The amp-iframe isn't displayed at all on first load. 

Expected beahviour is that the amp-iframe is displayed without any user actions.

**Comment by cathyxz** [comment]
The `<amp-iframe>` from your original code is in a tab that is shown via the `<amp-selector>` `toggle` action on `select` by the following code from the tab that is clicked. 

```
<amp-selector class="tabs-with-selector" 
role="tablist"
on="select:myTabPanels.toggle(index=event.targetOption, value=true)">
```

It's a duplicate of #20640, which will be fixed by #23688.

**Comment by romeuflores** [comment]
Whether it will fix our issue I can't be sure. 

But this is definitely the same as reported in #20640.

Whilst we do have a toggle action on our amp-selector, I'd like to stress the fact that the amp-iframe isn't displayed at all on first load. 

There are no toggle actions involved. Any tests shouldn't involve user actions which trigger the toggle event.
